.github/workflows/release.yml

```diff
@@ -14,7 +14,7 @@ jobs:
         with:
           fetch-depth: 0
       - name: Setup Node.js
-        uses: actions/setup-node@v2.4.1
+        uses: actions/setup-node@v2.5.0
```

Comment:
Maybe have this in another PR? :thinking:

Reply:
removed!